apps/condo/pages/ticket/index.tsx

```diff
@@ -186,70 +126,98 @@ export const TicketsPageContent = ({
+                                                                onChange={handleEmergencyChange}
+                                                                checked={emergency}
+                                                                style={CHECKBOX_STYLE}
+                                                            >
+                                                                {EmergenciesLabel}
+                                                            </Checkbox>
+                                                        </Col>
+                                                        <Col span={5}>
+                                                            <Checkbox
+                                                                onChange={handlePaidChange}
+                                                                checked={paid}
+                                                                style={CHECKBOX_STYLE}
+                                                            >
+                                                                {PaidLabel}
+                                                            </Checkbox>
+                                                        </Col>
+                                                    </Row>
+                                                </Col>
+                                                <Col>
+                                                    <Row>
+                                                        {
+                                                            appliedFiltersCount > 0 ? (
+                                                                <Col>
+                                                                    <Button
+                                                                        type={'text'}
+                                                                        onClick={resetQuery}
+                                                                    >
+                                                                        <Typography.Text strong type={'secondary'}>
+                                                                            {ResetFiltersLabel} <CloseOutlined style={CROSS_ICON_STYLE} />
+                                                                        </Typography.Text>
+                                                                    </Button>,
+                                                                </Col>
+                                                            ) : null
+                                                        }
+                                                        <Col>
+                                                            <Button
+                                                                secondary
+                                                                type={'sberPrimary'}
```

Comment:
we should use abstract types here like `primary`, `secondary` and so on

Reply:
sber prefix means branded color and behaviour, in several cases we cannot use abstract types